assets/src/edit-story/components/library/panes/media/common/mediaElement.js

```diff
@@ -54,11 +54,15 @@ const Video = styled.video`
   object-fit: cover;
 `;
 
-const Container = styled.div.attrs((props) => ({
+const Container = styled.button.attrs((props) => ({
```

Comment:
Would this have been reported as a violation by axe?
https://github.com/nickcolley/jest-axe

Reply:
Mmm not sure, probably not because it has no way of knowing it's meant to be clicked?